dtfs-central-api/src/types/tfm/tfm-facility.ts

```diff
@@ -0,0 +1,8 @@
+export type SpecificTfmFacilityValues = {
```

Comment:
Specific here doesn't really tell the reader any thing about why these fields and why this type exists. In any given scenario we will want to have access to different specific fields. 
In this scenario we want these specific values because they are the values required for keying sheet calculation so I would name this `KeyingSheetCalculationTfmFacilityValues`

Reply:
Agree, `Specific` is vague - in theory any function wants specific values 😛 

I'm not sure off the top of my head where this type is consumed but I recommend something simpler e.g `KeyingSheetFacilityValues`

Reply:
I really do want the word calculation in there since the keying sheet displays lots of values but these are specifically the facility values used for the utilisation and fixed fee adjustment calculations for the keying sheet so this really is already shortened!